code/modules/clothing/spacesuits/alien.dm

```diff
@@ -141,9 +143,217 @@
 		src.magpulse = 1
 		usr << "You dig your claws deeply into the flooring, bracing yourself."
 
+
 // Vox Casual
 // Civvie
-/obj/item/clothing/suit/space/vox/casual
+/obj/item/clothing/suit/space/vox/civ
+	name = "vox assistant pressure suit"
+	desc = "A cheap and oddly-shaped pressure suit made for vox crewmembers."
+	icon_state = "vox-civ-assistant"
+	item_state = "vox-pressure-normal"
+	body_parts_covered = UPPER_TORSO|LOWER_TORSO|LEGS|FEET|ARMS|HANDS
```

Comment:
There's really no need to copy all these flags.
They should be inherited from suit/space or suit/space/vox, and they make code harder to edit.